### PR TITLE
feat: added toast animation

### PR DESCRIPTION
**Pull request by abcrane123**
**What changed? Why?**
- adjust toast animation
- adjust toast message
- export toast icon component
**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: src/transaction/hooks/useGetTransactionStatus.tsx [hunk 1]
```diff
@@ -28,8 +28,10 @@ export function useGetTransactionStatus() {
       //   </a>
```
Comment (Zizzamia) [inline src/transaction/hooks/useGetTransactionStatus.tsx:28]: @abcrane123 can you remind me what's the story with this?

File: src/transaction/hooks/useGetTransactionToast.tsx [hunk 2]
```diff
@@ -62,6 +62,10 @@ export function useGetTransactionToast() {
       icon = <Spinner className="px-1.5 py-1.5" />;
       label = 'Transaction in progress';
     }
+    if (status === 'success') {
+      icon = successSVG;
+      label = 'Successful';
```
Comment (Zizzamia) [inline src/transaction/hooks/useGetTransactionToast.tsx:67]: All the SVGs let's move them into `internal/svg/` folder

File: site/tailwind.config.js [hunk 3]
```diff
@@ -30,6 +30,21 @@ export default {
       screens: {
         md: '848px',
       },
+      keyframes: {
```
Comment (Zizzamia) [inline site/tailwind.config.js:33]: I assume this is for test purposes?

File: src/transaction/hooks/useGetTransactionStatus.tsx (hunk 1 above)
Comment (abcrane123) [reply src/transaction/hooks/useGetTransactionStatus.tsx:28]: `useWriteContracts` returns a transaction id that looks something like `0xf595f3f5111f88b30c668946d4430cf74326aadec59fe846…0000000000000000000000000000000000000000000014a34`. i asked connor how i can use this id to get the correct block explorer link for a pending transaction. i pass this id to `useCallsStatus` to get the txn hash which i use to get the "view transaction" block explorer link but from the designs it looks like there is a way to get the explorer link with that transaction id.

File: site/tailwind.config.js (hunk 3 above)
Comment (abcrane123) [reply site/tailwind.config.js:33]: i wasn't sure where i need to put this to get the fade in animation to work. noticed there is a top level `tailwind.config.js` and this one. let me know if i should delete it.

Comment (Zizzamia) [reply site/tailwind.config.js:33]: ideally those should come from OnchainKit, no?

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**